source/dump.f90

```diff
@@ -573,7 +571,8 @@ subroutine dump_initialise
       end if
 
       ! Write format-specific headers
-      if (dumpfmt(i) == 1) then
+      call dump_getTasMatrix(i, tmpInv, tmpTas, tmpClo)
```

Comment:
What happens here for i=0, i=-1? (maybe add a small comment to remind)

Reply:
What happens for 0 and -1 depends on initialisation. Generally -1 is populated, and 0 is not.